src/main/webapp/resources/js/apis/requests.ts

```diff
@@ -0,0 +1,42 @@
+import axios, { AxiosRequestConfig } from "axios";
+
```

Comment:
Function comments

Reply:
There is a [separate PR](https://github.com/phac-nml/irida/pull/1332) in to handle this file and goes into a lot more depth, I would like to leave that in that PR if that is ok.

Reply:
That is acceptable :)